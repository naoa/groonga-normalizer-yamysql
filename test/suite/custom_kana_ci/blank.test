register normalizers/yamysql

normalize NormalizerYaMySQLKanaCISymbolPartofspeechHtml "クロアチア の戦争" WITH_TYPES|WITH_CHECKS
normalize NormalizerYaMySQLKanaCISymbolPartofspeechHtml " クロアチアの戦争" WITH_TYPES|WITH_CHECKS
normalize NormalizerYaMySQLKanaCISymbolPartofspeechHtml "クロアチアの戦争 " WITH_TYPES|WITH_CHECKS
normalize NormalizerYaMySQLKanaCISymbolPartofspeechHtml " クロアチアの戦争 " WITH_TYPES|WITH_CHECKS
normalize NormalizerYaMySQLKanaCISymbolPartofspeechHtml " クロアチア の戦争 " WITH_TYPES|WITH_CHECKS
normalize NormalizerYaMySQLKanaCISymbolPartofspeechHtml " クロアチア の戦争 " REMOVE_BLANK|WITH_TYPES|WITH_CHECKS
