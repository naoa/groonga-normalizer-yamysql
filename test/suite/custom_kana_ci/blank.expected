register normalizers/yamysql
[[0,0.0,0.0],true]
normalize NormalizerYaMySQLKanaCISymbolPartofspeechHtml "クロアチア の戦争" WITH_TYPES|WITH_CHECKS
[
  [
    0,
    0.0,
    0.0
  ],
  {
    "normalized": "クロアチア  戦争",
    "types": [
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "unknown",
      "unknown",
      "kanji",
      "kanji"
    ],
    "checks": [
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      1,
      3,
      3,
      0,
      0,
      3,
      0,
      0
    ]
  }
]
normalize NormalizerYaMySQLKanaCISymbolPartofspeechHtml " クロアチアの戦争" WITH_TYPES|WITH_CHECKS
[
  [
    0,
    0.0,
    0.0
  ],
  {
    "normalized": " クロアチア 戦争",
    "types": [
      "others",
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "unknown",
      "kanji",
      "kanji"
    ],
    "checks": [
      1,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      3,
      0,
      0,
      3,
      0,
      0
    ]
  }
]
normalize NormalizerYaMySQLKanaCISymbolPartofspeechHtml "クロアチアの戦争 " WITH_TYPES|WITH_CHECKS
[
  [
    0,
    0.0,
    0.0
  ],
  {
    "normalized": "クロアチア 戦争 ",
    "types": [
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "unknown",
      "kanji",
      "kanji",
      "unknown"
    ],
    "checks": [
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      3,
      0,
      0,
      3,
      0,
      0,
      1
    ]
  }
]
normalize NormalizerYaMySQLKanaCISymbolPartofspeechHtml " クロアチアの戦争 " WITH_TYPES|WITH_CHECKS
[
  [
    0,
    0.0,
    0.0
  ],
  {
    "normalized": " クロアチア 戦争 ",
    "types": [
      "others",
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "unknown",
      "kanji",
      "kanji",
      "unknown"
    ],
    "checks": [
      1,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      3,
      0,
      0,
      3,
      0,
      0,
      1
    ]
  }
]
normalize NormalizerYaMySQLKanaCISymbolPartofspeechHtml " クロアチア の戦争 " WITH_TYPES|WITH_CHECKS
[
  [
    0,
    0.0,
    0.0
  ],
  {
    "normalized": " クロアチア  戦争 ",
    "types": [
      "others",
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "unknown",
      "unknown",
      "kanji",
      "kanji",
      "unknown"
    ],
    "checks": [
      1,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      1,
      3,
      3,
      0,
      0,
      3,
      0,
      0,
      1
    ]
  }
]
normalize NormalizerYaMySQLKanaCISymbolPartofspeechHtml " クロアチア の戦争 " REMOVE_BLANK|WITH_TYPES|WITH_CHECKS
[
  [
    0,
    0.0,
    0.0
  ],
  {
    "normalized": "クロアチア 戦争",
    "types": [
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "unknown",
      "unknown",
      "kanji",
      "unknown"
    ],
    "checks": [
      4,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      4,
      3,
      0,
      0,
      3,
      0,
      0
    ]
  }
]
